charts/cass-operator-chart/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: Cass Operator
+version: 1.0.0
+description: Helm chart for Cass Operator.
+maintainers:
+  - name: Ryan Springer
+    email: ryan.springer@datastax.com
```

Comment:
I'm totally onboard with people emailing you when things don't work :-p

Reply:
oh, I forgot to add everyone.... ;)